src/components/FormFieldComponent.vue

```diff
@@ -298,15 +244,15 @@ export default {
         // Pass callback function to use entityMapper evaluation error handling response
         return scriptFunction(scriptData, (error) => {
           // Handle error comming back from eval function generated by the entityMapper
-          this.evalationError = error
+          this.evaluationError[this.id] = error
         })
       } catch (error) {
-        this.evalationError = error
+        this.evaluationError[this.id] = error
```

Comment:
I'm probably misunderstanding something, but doesn't a component have only 1 id? If so why is the evaluationError an object?

Reply:
That is true, I encountered some strange cross-component behaviour. using a scope fixed it.

Reply:
Maybe they took a bit too much inspiration from angularjs, there you had similar issues -_-